kratos/includes/model_part.h

```diff
@@ -11,42 +11,20 @@
 //
 //
 
-
-
 #if !defined(KRATOS_MODEL_PART_H_INCLUDED )
 #define  KRATOS_MODEL_PART_H_INCLUDED
 
-
-
 // System includes
-#include <string>
-#include <iostream>
-#include <sstream>
-#include <cstddef>
-
 
 // External includes
 
-
 // Project includes
-#include "includes/define.h"
 #include "includes/serializer.h"
 #include "includes/process_info.h"
-#include "containers/data_value_container.h"
-#include "includes/mesh.h"
```

Comment:
where is mesh included such that it also works without?
Could this be another cotire problem?

Reply:
Just checked, `communicator.h` includes (and uses) it